internal/connector/pingone_platform/resources/pingone_form.go

```diff
@@ -0,0 +1,71 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingoneFormResource{}
+)
+
+type PingoneFormResource struct {
+	clientInfo *connector.SDKClientInfo
+}
+
+// Utility method for creating a PingoneFormResource
+func Form(clientInfo *connector.SDKClientInfo) *PingoneFormResource {
+	return &PingoneFormResource{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingoneFormResource) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+
+	l.Debug().Msgf("Fetching all pingone_form resources...")
+
+	entityArray, response, err := r.clientInfo.ApiClient.ManagementAPIClient.FormManagementApi.ReadAllForms(r.clientInfo.Context, r.clientInfo.ExportEnvironmentID).Execute()
+	defer response.Body.Close()
+	if err != nil {
```

Comment:
Just thinking about reducing duplication from our meeting, maybe we could combine lines ~27-42 into a common method to share across our pingone resources? It might end up being more messy to fit it into something common than it's worth. Anyways, not something that has to be done for this review.

Reply:
That is a good thought. I'll look into adding it when we combine connector code with SSO 👍